app/services/case_remove_pit_extension_service.rb

```diff
@@ -0,0 +1,34 @@
+class CaseRemovePITExtensionService
+  attr_reader :result, :error
+
+  def initialize(user, kase)
+    @user = user
+    @case = kase
+    @result = :incomplete
+  end
+
+  def call
+    ActiveRecord::Base.transaction do
+      @case.state_machine.remove_pit_extension!(acting_user: @user,
+                                                acting_team: BusinessUnit.dacu_bmt)
+      @case.update external_deadline: (find_original_final_deadline)
+      @case.reload
+      @result = :ok
+    end
+    @result
+  rescue => err
+    Rails.logger.error err.to_s
+    Rails.logger.error err.backtrace.join("\n\t")
+    @error = err
+    @result = :error
```

Comment:
This will log details of the error, but give no other indication of when it happens.  Better to raise err and the end of the rescue block.